_episodes/06-plotting.md

```diff
@@ -0,0 +1,341 @@
+---
+title: "Plotting"
```

Comment:
Previously you mentioned you like longer titles. How about this
```suggestion
title: "Visualize your data by plotting it"
```

Reply:
Good point! But I'll keep the gerund form ("Visualizing ..."). Also, there's this thing about "data" being plural... :roll_eyes: